WcaOnRails/app/models/competition.rb

```diff
@@ -110,12 +111,16 @@ def create_id_and_cell_name
     if m
       name_without_year = m[1]
       year = m[2]
+      safe_name_without_year = ActiveSupport::Inflector.transliterate(name_without_year).gsub(/[^a-z0-9]+/i, '')
       if id.blank?
         # Generate competition id from name
-        # By replacing accented chars with their ascii equivalents, and then
+        # by replacing accented chars with their ascii equivalents, and then
         # removing everything that isn't a digit or a character.
```

Comment:
Nit: This comment looks out of place now that you moved `safe_name_without_year` up. Maybe move the comment as well?